src/Collapse.php

```diff
@@ -184,12 +184,22 @@ public function renderItem($header, $item, $index)
                 $header = Html::encode($header);
             }
 
+            $active = false;
+            if (isset($options['class'])) {
+                $classes = is_string($options['class']) ? explode(' ', $options['class']) : $options['class'];
```

Comment:
Doing it like that will behave well if one will use extra spaces while setting options:

```suggestion
                $classes = is_string($options['class']) ? preg_split('/\s+/', $options['class'], -1, PREG_SPLIT_NO_EMPTY) : $options['class'];
```